torch/testing/_internal/common_nn.py

```diff
@@ -108,6 +108,15 @@ def get_weight(m):
         with_tf32=True,
         tf32_precision=0.005,
     ),
+    dict(
+        module_name='Bias',
+        constructor_args=(10, ),
+        cpp_constructor_args='torch::nn::BiasOptions(10)',
+        input_size=(4, 10),
+        reference_fn=lambda i, p, _: i + p[0].view(1, -1).expand(4, 10),
+        with_tf32=True,
+        tf32_precision=0.005,
+    ),
```

Comment:
FYI these dicts are the old way of doing things, replaced by `ModuleInfo` entries in `torch/testing/_internal/common_modules.py`. It's not necessary to do for the purposes of this PR, just wanted to point out that the future is `ModuleInfo` :)